CMakeLists.txt

```diff
@@ -70,3 +70,5 @@ add_subdirectory(${QuokkaCode_SOURCE_DIR}/extern/yaml-cpp ${QuokkaCode_BINARY_DI
 add_definitions(-DNAUX_NET -DSTRANG)
 add_subdirectory(${QuokkaCode_SOURCE_DIR}/extern/Microphysics ${QuokkaCode_BINARY_DIR}/Microphysics)
 add_subdirectory(${QuokkaCode_SOURCE_DIR}/src ${QuokkaCode_BINARY_DIR}/src)
+
+execute_process(COMMAND python3 "${CMAKE_SOURCE_DIR}/scripts/quotes.py")
```

Comment:
You should just call `fortune`. It's on every Mac and Linux system:
```
FORTUNE(1)                                        General Commands Manual                                        FORTUNE(1)

NAME
       fortune - sample lines from a file

SYNOPSIS
       fortune [ file ]

DESCRIPTION
       Fortune prints a one-line aphorism chosen at random.  If a file is specified, the saying is taken from that file;
       otherwise it is selected from /lib/fortunes.

FILES
       /lib/fortunes
       /lib/fortunes.index   fast lookup table, maintained automatically

SOURCE
       /src/cmd/fortune.c
```